src/zenml/models/v2/core/event_source.py

```diff
@@ -48,7 +48,6 @@ class EventSourceRequest(WorkspaceScopedRequest):
     )
     plugin_subtype: PluginSubType = Field(
         title="The plugin subtype of the event source.",
-        max_length=STR_FIELD_MAX_LENGTH,
```

Comment:
As I mentioned in the description, we ran into an issue with the deployment of our local server when our models featured `Enum` fields that were parameterized with a `max_length`. Since the Enums are "length-wise" in our control, I removed the flags for now.